README.md

```diff
@@ -49,6 +49,14 @@ If you want to build/install shared libraries for AV1 codecs, you can still
 peek inside of each script to see where the current known-good SHA is for each
 codec.
 
+### Tests
+
+A few tests can be built by enabling the `AVIF_BUILD_TESTS` CMake option.
+
+The remaining tests can be built by enabling the `AVIF_BUILD_TESTS` and
+`AVIF_BUILD_GTESTS` CMake options. They require GoogleTest to be built locally
```

Comment:
should AVIF_BUILD_GTESTS automatically set AVIF_BUILD_TESTS?

Reply:
It could be useful to still have a way to at least run `aviftest` in environments where GoogleTest is unavailable.

I would suggest to set `AVIF_BUILD_GTESTS` to `ON` by default instead. This way enabling `AVIF_BUILD_TESTS` would be enough to run all tests. Another option would be a `AVIF_BUILD_NO_GTEST` flag but it seems bad.

Reply:
> It could be useful to still have a way to at least run `aviftest` in environments where GoogleTest is unavailable.
> 
> I would suggest to set `AVIF_BUILD_GTESTS` to `ON` by default instead. This way enabling `AVIF_BUILD_TESTS` would be enough to run all tests. Another option would be a `AVIF_BUILD_NO_GTEST` flag but it seems bad.

I agree NO isn't the best. I was only thinking that since AVIF_BUILD_GTESTS requires AVIF_BUILD_TESTS, cmake -DAVIF_BUILD_GTESTS=1 could be a shorthand for cmake -DAVIF_BUILD_GTESTS=1 -DAVIF_BUILD_TESTS=1